.github/workflows/backend-test.yml

```diff
@@ -10,9 +10,10 @@ on:
       - main
       - rc-*
       - testing-rc-*
-    
+
 permissions:
   contents: read
+  pull-requests: write
```

Comment:
btw. This permission should only be in the specific job below rather than at the top. Otherwise it gives permissions to all of them. Even if there aren't others now, there could be later. This is checked by the openssf scorecard https://scorecard.dev/viewer/?uri=github.com/headlamp-k8s/headlamp

More details in the docs here: https://github.com/ossf/scorecard/blob/7ce8609469289d5f3b1bf5ee3122f42b4e3054fb/docs/checks.md#token-permissions